packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -219,6 +219,18 @@ export abstract class SelectDirPath
     const rootPath = vscode.workspace.rootPath;
     let outputdir;
     if (rootPath) {
+      if (
```

Comment:
@lcampos @vazexqi Is this how we handle this sort of error normally? I would have assumed we would let the CLI surface the error message.

Reply:
Normally we rely on the cli to surface errors but in this specific scenario this code involves the extensions directly reading the file system to determine what data is rendered in showQuickPick UI (e.g. the dir list shown when running `SFDX: Create Apex Class`. This way of handling/surfacing an error is correct.

![screen shot 2018-11-30 at 1 43 08 pm](https://user-images.githubusercontent.com/1041842/49316526-0fb3f300-f4a6-11e8-8c15-2f8b0f15eb32.png)